components/list/list.css

```diff
@@ -0,0 +1,57 @@
+.list {
+    width: 20rem;
+}
+
+.list-group {
+    list-style: none;
+}
+
+.list-group-item {
+    padding: 10px 25px;
+    border: 1px solid white;
+    font-size: medium;
+}
+
+.active {
+    background-color: grey
+}
+
+.disabled {
+    color: grey;
+}
+
+.list-item-dark {
+    color: black;
+}
+
+.list-group-item-primary {
+    background-color: lightblue;
+}
+
+.list-group-item-secondary {
+    background-color: lightgrey;
+}
+
+.list-group-item-success {
+    background-color: lightgreen;
+}
+
+.list-group-item-danger {
```

Comment:
You can use color variables for all the background colors mentioned and also rem